src/class-parsely.php

```diff
@@ -1623,7 +1625,7 @@ private function get_categories( $post_id, $delimiter = '/' ) {
 	 * options are saved, they override the defaults.  This prevents us from having to do a lot of isset() checking
 	 * on variables.
 	 */
-	private function get_options() {
+	public function get_options() {
```

Comment:
@hbbtstar @GaryJones -- any qualms about making this public? Can you think of a better way to get this data in the endpoint here?

https://github.com/Parsely/wp-parsely/blob/171714f56d90a57f8385dfd74965d775fc5846cc/src/settings-api.php#L21

Reply:
Not at this point, no. In a future iteration where the Parsely class didn't exist, then we might be more careful about how we share and access settings, but for this PR, go ahead and make it public.